app/scripts/views/ready.js

```diff
@@ -9,18 +9,54 @@
 
 define([
   'cocktail',
-  'views/form',
-  'stache!templates/ready',
-  'lib/url',
   'lib/constants',
-  'views/mixins/service-mixin',
+  'lib/url',
+  'stache!templates/ready',
+  'views/form',
   'views/marketing_snippet',
-  'views/marketing_snippet_ios'
+  'views/marketing_snippet_ios',
+  'views/mixins/service-mixin'
 ],
-function (Cocktail, FormView, Template, Url, Constants, ServiceMixin,
-      MarketingSnippet, MarketingSnippetiOS) {
+function (Cocktail, Constants, Url, Template, FormView, MarketingSnippet,
+  MarketingSnippetiOS, ServiceMixin) {
   'use strict';
 
+  function t(msg) {
+    return msg;
+  }
+
+  function generateHeader (id, text) {
+    return '<h1 id="' + id + '">' + text + '</h1>';
+  }
+
+  /*eslint-disable camelcase*/
+
+  // The "header" and "ready to sync" messages are generated
```

Comment:
@vladikoff - is this any better? The template is definitely cleaner, but now the JS is smelly. 

Other alternatives:
* One template per screen - pro - fewer switches per screen. con w/o native template support for `layout`, lots of duplicate HTML across templates. My attempt at a homegrown `layout` solution was ugly.
* One module per screen with the header HTML and ready to sync text as properties of the module. Pro - each module is relatively clean. Con - explosion of modules to take care of strings.
* Pass the strings in from router.js as configuration. Pro - ready.js is clean. Con - router.js starts to become dirty.

Thoughts?